### PR TITLE
Glue benchmark

### PR DESCRIPTION
**Pull request by leonardozcm**

### CODE REVIEW

File: examples/pytorch/eager/huggingface_models/examples/text-classification/Benchmark.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+pruning takes time > 15h
+
+```
+python examples/text-classification/run_glue_no_trainer_prune.py --task_name mnli --max_length 128 \
+       --model_name_or_path Intel/bert-base-uncased-sparse-70-unstructured \
+       --per_device_train_batch_size 32 --learning_rate 5e-5 --num_train_epochs 3 --output_dir /tmp/$TASK_NAME/ \
+       --prune --config prune.yaml --output_model prune_model/model.pt --seed 5143
+```
+
+## ONNX:
```
Comment (yangw1234) [inline examples/pytorch/eager/huggingface_models/examples/text-classification/Benchmark.md:47]: does this case use dynamic quantization? https://huggingface.co/transformers/serialization.html#quantization

File: examples/pytorch/eager/huggingface_models/examples/text-classification/Benchmark.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+export TASK_NAME=MRPC
+python examples/text-classification/run_glue_tune.py  --model_name_or_path roberta-base   --task_name $TASK_NAME   --do_train  --do_eval  --max_seq_length 128  --per_device_train_batch_size 32   --learning_rate 2e-5   --output_dir /tmp/$TASK_NAME/ --overwrite_output_dir
+
+```
+Alternately, you could also specify --model_name_or_path to the directory of local .bin model to skip training.
+```
+python examples/text-classification/run_glue_tune.py  --model_name_or_path /tmp/MRPC   --task_name $TASK_NAME   --do_eval  --max_seq_length 128  --per_device_train_batch_size 32   --learning_rate 2e-5   --output_dir /tmp/$TASK_NAME/ --overwrite_output_dir
+```
+
+4 pinned Multiprocessing:
```
Comment (yangw1234) [inline examples/pytorch/eager/huggingface_models/examples/text-classification/Benchmark.md]: can we also try this on bigdl nano?
